analyzers/tests/SonarAnalyzer.UnitTest/Extensions/SafeVisualBasicSyntaxWalkerTest.cs

```diff
@@ -29,6 +29,8 @@ public class SafeVisualBasicSyntaxWalkerTest
         public void GivenSyntaxNodeWithReasonableDepth_SafeVisit_ReturnsTrue() =>
             new Walker().SafeVisit(SyntaxFactory.ParseSyntaxTree("Public Function Main(Arg as Boolean) As Boolean").GetRoot()).Should().BeTrue();
 
+#if NET
+
         [TestMethod]
         public void GivenSyntaxNodeWithHighDepth_SafeVisit_ReturnsFalse()
```

Comment:
Same here

Reply:
Also to be part of the investigation background task.